### ISSUE TITLE
React App fails to load in the browser

### ISSUE THREAD

**Issue by nbkhope** [nbkhope/weather-sup#4]
The following error appears when you try loading the page:

```
Uncaught TypeError: Super expression must either be null or a function, not undefined
```

**Comment by nbkhope** [comment]
The problem is with the package `react-sparklines@1.7.0`. That version is broken. Downgrade to 1.6 to fix the issue:

```
npm install react-sparklines@1.6
```

Ref https://github.com/borisyankov/react-sparklines/issues/89

**Comment by nbkhope** [comment]
Fixed in 1648d39268696b849033cebb0c130b74e9a795da
